client/controller/models/apps/apps.go

```diff
@@ -97,15 +97,12 @@ func Logs(c *client.Client, appID string, lines int) (string, error) {
 
 	body, err := c.BasicRequest("GET", u, nil)
 
-	if err != nil {
-		return fmt.Sprintf("Error:%v", err), err
```

Comment:
No the problem here is that the err is almost assuredly a status code error from making the http call. That can be for a variety of reasons. If we return err we basically get output like

```
Error:
Status 500 ....
```

Its not helpful to the end user. Instead I give them a response that tells them what to check for.